webapp/src/components/expanded_view/component.tsx

```diff
@@ -229,6 +229,7 @@ export default class ExpandedView extends React.PureComponent<Props, State> {
                     emojiVersion={EMOJI_VERSION}
                     skinTonePosition='search'
                     onEmojiSelect={this.handleUserPicksEmoji}
+                    onClickOutside={this.toggleEmojiPicker}
```

Comment:
Does this handler fire only when the user clicks outside *and* the emoji picker is open? Or also when it's closed?

Reply:
Nevermind, I tested it and it works great!

Reply:
Only when it's open since the picker is already wrapped in an if check for `showEmojiPicker`